MekHQ/src/mekhq/gui/utilities/StaticChecks.java

```diff
@@ -4,7 +4,9 @@
 import java.util.UUID;
 import java.util.Vector;
 
+import megamek.common.BattleArmor;
```

Comment:
Why add these imports?

Reply:
Weird, the changes looked like they had been reverted when i first opened this PR. They were part of one stage of this branch, but I ended up reverting them

Reply:
Reverted